nuxt.config.js

```diff
@@ -5,27 +5,37 @@ export default {
 	mode: "universal",
 	generate: {
 		fallback: true,
-		routes (callback) {
+		async routes (callback) {
+			// Determine how many pages of posts are available
+			const totalPages = await (await axios.get(`${Config.wpDomain}${Config.apiBase}posts`)).headers["x-wp-totalpages"]
+			// Create empty array to hold all retrieved post data in chunks of 10
+			const postPagesAPI = []
+			// Create empty array to hold all pagination routes for the News and Views page (e.g. /page/1, /page/2, etc)
+			const newsPaginationRoutes = []
+			for (let postPage = 1; postPage <= totalPages; postPage++) {
+				// Add each page of posts to postPagesAPI
+				postPagesAPI.push(axios.get(`${Config.wpDomain}${Config.apiBase}posts?page=${postPage}`))
+				// Add each page index to newsPaginationRoutes
+				newsPaginationRoutes.push({ route: `/news-and-views/page/${postPage}`, payload: postPage })
+			}
+
+			// Spread the entries in postPagesAPI and fetch each post
 			axios.all([
-				axios.get(`${Config.wpDomain}${Config.apiBase}pages`),
-				axios.get(`${Config.wpDomain}${Config.apiBase}posts`)
+				...postPagesAPI
 			])
-				.then(axios.spread(function (pages, posts) {
-					const pageRoutes = pages.data.map((page) => {
-						return {
-							route: (page.slug !== "home") ? "/" + page.slug : "/",
-							payload: page
-						}
-					})
-
+				.then(axios.spread((posts) => {
+					// Add each post to the postRoutes variable
+					// replace line below with this when totalPages > 1:
+					// const postRoutes = posts.data.map((...post) => {
 					const postRoutes = posts.data.map((post) => {
 						return {
 							route: "/news-and-views/" + post.slug,
 							payload: post
 						}
 					})
 
-					callback(null, pageRoutes.concat(postRoutes))
+					// What happened to the page routes (for /about, /tools, etc.)?
```

Comment:
I noticed that your latest changes to the Nuxt config remove the code which sets up the routes for pages. Is there a reason for this?

Reply:
Yeah, the generate field should be used to build the dynamic pages only (pages where the parent folder name starts with underscore only, e.g. _webpage). All other regular pages are built automatically without specifying them in the generate field of nuxt.config.

Reply:
I think we should be generating the regular pages dynamically as well, but that's out of the scope of this pull request.

Reply:
They still are generated without specifying them in the generate field. You can test this out by running `npm run generate`. You can see the regular pages are still generated statically without specifying them

Reply:
How does that happen?

Reply:
`npm run generate` automatically builds all pages statically excluding the dynamic pages. Those have to be specified in the generate field of nuxt.config

Reply:
How does it automatically build all pages statically, is my question?